pkg/utils/utils.go

```diff
@@ -220,39 +226,31 @@ func ReplaceAllContainerImageApply(img K8sImagesConfig, c *acorev1.ContainerAppl
 	case csmv1.Externalhealthmonitor:
 		*c.Image = img.Images.Externalhealthmonitor
 	case csmv1.Sdc:
-		*c.Image = img.Images.Sdcmonitor
+		*c.Image = img.Images.Sdc // DISCUSS: Why does this point to Sdcmonitor and not Sdc?
```

Comment:
Is this comment needed? Seems like it was pointing to sdcmonitor by mistake

Reply:
I don't know for sure, since it was definitely changed intentionally and was *previously* Sdc before changing to SdcMonitor. 

@harishp8889 https://github.com/dell/csm-operator/commit/3074c0c92248197da5e1ddcfffc0c1bf7e16ba28#diff-81ddbadfb415ccbb9c7af84f11668d1aa5e53c34025bf86d4702f16b4e42f045R218 
Was there a reason this was changed to SdcMonitor in this commit/PR?

Reply:
@ChristianAtDell @shaynafinocchiaro This default.yaml has only sdcmonitor entry which points to sdc image. https://github.com/dell/csm-operator/blob/5bc17bd8a2a6d308f8e8383dd49242b432b858c7/operatorconfig/driverconfig/common/default.yaml#L21
Bcoz of this, the line should be ***c.Image = img.Images.Sdcmonitor**

Reply:
This is one of those places where a comment is useful to highlight something not obvious nor intuitive.

Reply:
changed the line and added a comment.